packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -201,3 +201,27 @@ test('sort alerts', async () => {
   });
   expect(response.items.length).toBeGreaterThan(0);
 });
+
+test('cursor pagination', async () => {
+  const client: CogniteClient = setupLoggedInClient();
+  const response = await client.alerts.list({
+    limit: 1,
+    sort: {
+      property: 'createdTime',
+      order: 'desc',
+    },
+    cursor: '',
+  });
+  expect(response.items.length).toBe(1);
+  // test for more than 1000 items
+  const response2 = await client.alerts.list({
+    limit: 1001,
```

Comment:
limits is always per page on all other resources on CDF.
to get more then 1000, user would use `autoPagingToArray()`.
check how it's done in other resources, e.g.
```
const timeseries = await response.autoPagingToArray({ limit: 5000 });
```